src/database/migrations/create-5-rooms.js

```diff
@@ -0,0 +1,39 @@
+export default {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('rooms', {
+    id: {
+      allowNull: false,
+      primaryKey: true,
+      type: Sequelize.UUID,
+      defaultValue: Sequelize.literal('uuid_generate_v4()')
+    },
+    type: {
```

Comment:
What information is `type` expected to contain?

Reply:
the type of accommodation rooms available. I got it from this related pivotal tracker story [#167750075](https://www.pivotaltracker.com/story/show/167750075)